### PR TITLE
ADEN-1840 Add ads mobile hop tests

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsHopObject.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  private static final String POST_MESSAGE_SCRIPT_XPATH =
+      "//script[contains(text(), 'parent.postMessage')]";
+
+  public AdsHopObject(WebDriver driver, String page) {
+    super(driver, page);
+  }
+
+  public AdsHopObject verifyClassHidden(String slotName, int testedDivNumber) {
+    WebElement testedDiv = getTestedDiv(slotName, testedDivNumber);
+    Assertion.assertEquals(" hidden", testedDiv.getAttribute("class"));
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsHopObject.java]: Why ` hidden` not `hidden`? :japanese_ogre:

File: src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java [hunk 2]
```diff
@@ -9,119 +9,127 @@
...
+            "wka.ent/_adtest//article",
+        }
+    };
+  }
+
+  @DataProvider
+  public static Object[][] testAdsHopPostMessage() {
+    return new Object[][]{
+        {"adtest", "DFP/hop", 1, "mobile"},
+        {"adtest", "DFP/hophop", 2, "mobile_remnant"}
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java]: Consider removing magic numbers and using just source to get the the right div you want.

**Comment by nandy-andy** [comment]
Two notes left. We can discuss it after daily because it doesn't make sense to write novels here :japanese_ogre:

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsHopObject.java (hunk 1 above)
Comment (drets) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsHopObject.java]: Surely. I think it's a defect. Why we need to have extra whitespace?

File: src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java (hunk 2 above)
Comment (drets) [inline src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java]: Done.

**Comment by nandy-andy** [comment]
LGTM

**Comment by nandy-andy** [comment]
:+1:

**Pull request merged**